bin/od

```diff
@@ -24,9 +24,9 @@ use constant LINESZ => 16;
 use constant PRINTMAX => 126;
 
 use vars qw/ $opt_A $opt_a $opt_B $opt_b $opt_c $opt_d $opt_e $opt_F $opt_f
- $opt_i $opt_j $opt_l $opt_N $opt_o $opt_s $opt_v $opt_x /;
+ $opt_H $opt_i $opt_j $opt_l $opt_N $opt_o $opt_s $opt_v $opt_X $opt_x /;
 
-our $VERSION = '1.0';
+our $VERSION = '1.1';
```

Comment:
We might consider a big bump here since there are new features. But, since the new features don't interfere with current behavior, a minor bump is fine too.